Python/pywarpx/picmi.py

```diff
@@ -489,12 +489,30 @@ def initialize_inputs(self):
         self.laser.phi2 = self.phi2
 
 
+class AnalyticLaser(picmistandard.PICMI_AnalyticLaser):
+    def initialize_inputs(self):
+        self.laser_number = pywarpx.lasers.nlasers + 1
+        self.name = 'laser{}'.format(self.laser_number)
```

Comment:
I know that this is already in the original picmi file.
But isn't the laser name set by the user?
Also (just for curiosity) why is it better to make a new laser with a string name 'laser1' or 'laser2', etc. than an array of lasers or just making the new laser with the number directly, 1, 2 etc.?

Reply:
Currently, in PICMI, there is no name option for lasers, but it could be added. Good idea!
It is given a string name so that it can have a unique identifier in the WarpX input.

Reply:
Thank you for the answer. I understand it now. It is ok as it is as well. I was just confused, but it makes sense now, that this is necessary for compatibility with WarpX.

Reply:
I've added the name option to picmi and allow it here.

Reply:
Thanks for adding it!